static/src/javascripts/projects/common/modules/experiments/tests/auxia-sign-in-gate.js

```diff
@@ -0,0 +1,23 @@
+
+export const auxiaSignInGate = {
+	id: 'AuxiaSignInGate',
+	start: '2025-01-23',
+	expiry: '2026-01-30',
+	author: 'Pascal (Growth Team)',
+	description: 'R&D Experiment: using Auxia API to drive the behavior of the SignIn gate',
+	audience: 0.01,
+	audienceOffset: 0,
+	successMeasure: '',
+	audienceCriteria: '',
+	ophanComponentId: 'auxia_signin_gate',
+	dataLinkNames: 'AuxiaSignInGate',
+	idealOutcome: '',
+	showForSensitive: false,
+	canRun: () => true,
+	variants: [
+		{
+			id: 'auxia-signin-gate',
+			test: () => {},
+		},
+	],
```

Comment:
Again, just confirming, do you want a control and a variant here, so you'd be able to measure the impact of variant in a fair way?

Reply:
We are not measuring impact for the moment. It's just a proof of concept. 
@SiAdcock

Reply:
@SiAdcock 
I defined the audience to zero here also :)